apis/core/v1alpha1/restoresession_types.go

```diff
@@ -177,6 +177,15 @@ type MSSQLServerManifestOptions struct {
 	// +optional
 	AuthSecretName string `json:"authSecretName,omitempty"`
 
+	// Archiver specifies whether to restore the Archiver manifest or not
+	// +kubebuilder:default=true
+	// +optional
+	Archiver *bool `json:"archiver,omitempty"`
+
+	// ArchiverRef specifies the name of the ArchiverRef
```

Comment:
// ArchiverRef specifies the new name and namespace of the Archiver yaml after restore